resources/views/welcome.blade.php

```diff
@@ -167,38 +167,37 @@
             </div>
         @endif
         @if($signedLatestAgreement && !$needsTransaction && !$needsPayment)
-        @if($needTravelPayment)
-            <div class="col-sm-6 com-md-3 col-lg-4">
-                <div class="card">
-                    <div class="card-body">
-                        <h4 class="card-title">
-                            Travel Payment Required
-                        </h4>
-                        <p class="card-text">
-                            You need to make a payment for {{ $travelName }}.
-                        </p>
-                        <a href="{{ route('pay.travel') }}">Pay Online Now</a> or <a href="{{ route('travel.index') }}">View Travel</a>
+            @if($needTravelPayment)
+                <div class="col-sm-6 com-md-3 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h4 class="card-title">
+                                Travel Payment Required
+                            </h4>
+                            <p class="card-text">
+                                You need to make a payment for {{ $travelName }}.
+                            </p>
+                            <a href="{{ route('pay.travel') }}">Pay Online Now</a> or <a href="{{ route('travel.index') }}">View Travel</a>
+                        </div>
                     </div>
                 </div>
-            </div>
-        @endif
-        @if($needTravelAuthorityRequest)
-            <div class="col-sm-6 com-md-3 col-lg-4">
-                <div class="card">
-                    <div class="card-body">
-                        <h4 class="card-title">
-                            Travel Authority Request Required
-                        </h4>
-                        <p class="card-text">
-                            You need to submit a Travel Authority Request for {{ $travelName }}.
-                        </p>
-                        <a href="{{ $travelAuthorityRequestUrl }}">Submit Now</a> or <a href="{{ route('travel.index') }}">View Travel</a>
+            @endif
+            @if($needTravelAuthorityRequest)
+                <div class="col-sm-6 com-md-3 col-lg-4">
+                    <div class="card">
+                        <div class="card-body">
+                            <h4 class="card-title">
+                                Travel Authority Request Required
+                            </h4>
+                            <p class="card-text">
+                                You need to submit a Travel Authority Request for {{ $travelName }}.
+                            </p>
+                            <a href="{{ $travelAuthorityRequestUrl }}">Submit Now</a> or <a href="{{ route('travel.index') }}">View Travel</a>
+                        </div>
                     </div>
                 </div>
-            </div>
-        @endif
+            @endif
         @endif
+        <self-service-access-override />
```

Comment:
Is there a particular reason this is rendered client side?

Reply:
It enables the swal dialog to show info after requesting the override.  It is theoretically possible for this to be server-side, but I think the UX is smoother this way.  I could probably clean up some of the dashboard logic related to overrides since I didn't end up using that.